src/test/java/net/snowflake/client/log/JDK14LoggerWithClientLatestIT.java

```diff
@@ -26,7 +26,7 @@ public void testJDK14LoggingWithClientConfig() {
     try {
       Files.write(configFilePath, configJson.getBytes());
       Properties properties = new Properties();
-      properties.put("clientConfigFile", configFilePath.toString());
+      properties.put("client_config_file", configFilePath.toString());
```

Comment:
why don't we use `SFSessionProperty.CLIENT_CONFIG_FILE.getPropertyKey()` in all those tests?

Reply:
This is how customers are going to use the property name. This integration test resembles how customers are going to use the connection property. If we accidentally change the property name, this test will fail and block the change.